### PR TITLE
SAN-152 - Updates to web security and bump parent pom version to latest version

### PR DESCRIPTION
**Pull request by dgordon-ch**

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java [hunk 1]
```diff
@@ -9,62 +9,62 @@
...
     }
 
     @Bean
     @Order(4)
-    protected SecurityFilterChain healthcheckSecurityFilterChain(HttpSecurity http) throws Exception {
-        http
-                .securityMatcher("/late-filing-penalty/healthcheck");
-        return http.build();
+    public SecurityFilterChain healthcheckSecurityFilterChain(final HttpSecurity http) throws Exception {
+        return configureWebCsrfMitigations(
```
Comment (rbull-ch) [inline src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java:47]: Is this one supposed to be `configureApiCsrfMitigations`?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java (hunk 1 above)
Comment (dgordon-ch) [reply src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java:47]: Don't believe so, the api one doesn't add the session handler or csrf filter (I know we're not defining the session handler specifically in ppsWebSecurityFilterConfig anymore). But, we can try it if this one fails...

Comment (dgordon-ch) [reply src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java:47]: Actually, going to change it, makes sense, will not change accessibility-statement as we at least have a controller for that.

**Review by rbull-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**